src/cdk/v2/destinations/the_trade_desk/utils.js

```diff
@@ -0,0 +1,130 @@
+const lodash = require('lodash');
+const CryptoJS = require('crypto-js');
+const jsonSize = require('json-size');
+const { InstrumentationError, AbortedError } = require('@rudderstack/integrations-lib');
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  getSuccessRespEvents,
+  removeUndefinedAndNullValues,
+  handleRtTfSingleEventError,
+} = require('../../../../v0/util');
+const {
+  DATA_PROVIDER_ID,
+  MAX_REQUEST_SIZE_IN_BYTES,
+  DATA_SERVERS_BASE_ENDPOINTS_MAP,
+} = require('./config');
+
+const ttlInMin = (ttl) => parseInt(ttl, 10) * 1440;
+const getBaseEndpoint = (dataServer) => DATA_SERVERS_BASE_ENDPOINTS_MAP[dataServer];
+const getFirstPartyEndpoint = (dataServer) => `${getBaseEndpoint(dataServer)}/data/advertiser`;
+
+const getSignatureHeader = (request, secretKey) => {
+  if (!secretKey) {
+    throw new AbortedError('Secret key is missing. Aborting');
+  }
+  const sha1 = CryptoJS.HmacSHA1(JSON.stringify(request), secretKey);
+  const base = CryptoJS.enc.Base64.stringify(sha1);
+  return base;
+};
+
+const responseBuilder = (items, Config) => {
```

Comment:
we don't generally name args name starting with Capital letters

Reply:
updated, used **config**